### DISCUSSION TITLE
Сделать счетчик подсчета высоты

### DISCUSSION THREAD

**Discussion by thempra79** [sakuraskip/opiProject12#1] [Ideas]
Я написал код для счетчика в проект, осталось только подключить шрифт.

**Comment by sakuraskip** [comment]
Бро идея просто сумасшедшая осталось только заставить работать!!!!!!!!

**Reply by desynchrome** [reply]
кого???????

**Comment by desynchrome** [comment]
парни ну идея реально класс но блин а как эти рисовашки сделать-то?

**Comment by sakuraskip** [comment]
все бля вы меня надоели
